add_me_to_your_PATH/sysimage.jl

```diff
@@ -13,8 +13,9 @@ if project.ispackage && !isfile(modulefile)
     end
 end
 
-# Skip generating a system image when there are no dependencies
-isempty(project.dependencies) && exit(0)
+# Skip generating a system image when there are no dependencies,
+# or all deps are stdlibs.
+isempty(project.dependencies) || (values(project.dependencies) ⊆ keys(Pkg.Types.stdlibs())) && exit(0)
```

Comment:
This doesn't do what the comment states. Example:
```julia
julia> true || false && println("hi")
true
```
You want:
```suggestion
if isempty(project.dependencies) || values(project.dependencies) ⊆ keys(Pkg.Types.stdlibs())
    exit(0)
end
```
```

Reply:
oops, yup, thanks!